### DISCUSSION TITLE
Calendar: how to get a printable or minimal view

### DISCUSSION THREAD

**Discussion by daks** [pretalx/pretalx#1891] [Q&A] [answered]
Hello,
I'm going to an event which uses pretalx and I would like to print the calendar to make my own selection and be able to navigate easily.
The thing is that it has soomething like 15 rooms (columns) and 10 rows (by half an hour) per day which makes the calendar not printable as PDF by default: everything is too small to be usable on a A4 paper.

Is there anyway to print/output it in a text format?

**Comment by rixx** [comment]
You could use the "sessions" tab, which is just a list of sessions, but ordered chronologically. If you star sessions in pretalx, you can also filter your schedule to only show your starred sessions, which should also cut down on the paper used.

If you really just want a plain-text list, you can also just `curl` the schedule page, and append `?format=list` (the default is a grid, but that seems not practical in any normal terminal), which will give you a very plain list of sessions / dates.

**Reply by daks** [reply]
Sadly I don't see any session tab here https://capitoledulibre.org/programme/

I also tried `curl https://capitoledulibre.org/programme/` without success. I think it complains about lack of javascript :(

**Reply by rixx** [reply, marked as answer]
Ah, well – on that page, you only see the embedded pretalx plugin. The sessions tab and the `curl` command both referred to the actual pretalx page. I'm afraid the embedded widget will not support printing at any point (though we are going to add a way to hide rooms eventually, which should help with the printing). As the embedded widget cannot control the surrounding site, there’s just too much we can’t influence there, so working on print support there is near-impossible.

**Comment by calendarforum392-arch** [comment]
Printing pretalx schedules can be challenging since the default grid view is interactive. A simpler approach is using the “Sessions” list view or the ?format=list URL parameter to get a clean, chronological export. For those who want a printable schedule or a minimal calendar view, tools like Calendar-Vibe
 can help create easy-to-print calendars from session lists.

**Comment by rixx** [comment]
The embed widget at this points permits the embedding site to select which rooms should be shown, so you could build multiple schedules and only print as many rooms as will fit a page.
